ietf/nomcom/utils.py

```diff
@@ -165,6 +166,8 @@ def delete_nomcom_templates(nomcom):
     nomcom_template_path = '/nomcom/' + nomcom.group.acronym
     DBTemplate.objects.filter(path__contains=nomcom_template_path).delete()
 
+def command_line_safe_secret(secret):
+    return base64.encodebytes(secret).decode('utf-8')[:-2]
```

Comment:
The `[:-2]` is chopping off the base64 padding, is that desirable? If the secret length changes, this might chomp actual bytes (or leave stray padding characters).

Reply:
it should be a -1 I suppose - the intent was to get the \n.

Reply:
Makes sense - maybe use rstrip() instead?